tracer/src/Datadog.Trace/RemoteConfigurationManagement/RemoteConfigurationManager.cs

```diff
@@ -150,47 +150,42 @@ public static void CallbackWithInitializedInstance(Action<RemoteConfigurationMan
             action(inst);
         }
 
-        public async Task StartPollingAsync()
+        public Task StartPollingAsync()
```

Comment:
So this doesn't need to be async any more right? Should we just make it `StartPolling()` and ditch the `Task`?

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/4076/commits/66885c6ecd0ae6822191e868dedfb49b127d5c3f